### PR TITLE
Fixes #2155: Refactor thanks.jst to use new design and styles.

### LINKED ISSUE #2155: Layout becomes messed up after reporting an issue
URL: https://staging.webcompat.com/
Browser / Version: Firefox Nightly Mobile 60.0a1 (2018-02-18), Firefox Nightly Desktop 60.0a1 (2018-02-16), Chrome Desktop 64.0.3282.167, Chrome Mobile 64.0.3282.137 
Operating System: Windows 10 Pro, Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Steps to Reproduce:**
1. Navigate to https://staging.webcompat.com/
2. Tap "Report bug" button.
3. Type in all required information.
4. Tap "Report" button (anonymous of with the Github account).
5. Observe behavior after issue is logged successfully.

**Expected Behavior:**
An overlay with "Thanks" message is displayed and dismissed for after a few seconds.
"Share on Twitter"/"Share on Facebook" button are displayed under the reported issue.

**Actual Behavior:**
"Thanks for reporting an issue!" message overlaps the bug report.

**Note**
1. Workaround: Refreshing the screen, the "Thanks" message is dismissed.
2. "Thanks" message: 
```html
<div class="notification-bar js-flashmessage is-active notification-thanks">
	<h4>Thanks for reporting an issue!</h4>
	<p>You're helping us make the web a better place to work and play.</p>
	<p>Tell your friends about the bug you just filed:</p>
	<a class="wc-Button wc-Button--action" href="https://twitter.com/intent/tweet?text=I%20just%20filed%20a%20bug%20on%20the%20internet%3A&amp;url=https%3A%2F%2Fwebcompat.com%2Fissues%2F1376&amp;via=webcompat" target="_blank">Share on Twitter</a>
	<a class="wc-Button wc-Button--action" href="https://facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwebcompat.com%2Fissues%2F1376" target="_blank">Share on Facebook</a>
</div>
```

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

screenshot-1519044666800

thanksmessageoverbugreport

### PR DESCRIPTION
**Pull request by miketaylr**
I'm sure it could be refined further... but this seems good enough to ship (not uglier than the current production version anyways :p)

screen shot 2018-03-16 at 4 17 37 pm

r? @zoepage

### CODE REVIEW

File: webcompat/static/css/src/variables.css [hunk 1]
```diff
@@ -14,6 +14,8 @@
   --unit-modul-l: calc(910px - var(--unit-space)); /* x3 grid unit */
   --unit-modul-m: calc(2 / 3 * var(--unit-modul-l) - var(--unit-space)); /* x2 grid unit */
   --unit-modul-s: calc(1 / 3 * var(--unit-modul-l) - var(--unit-space)); /* x1 grid unit */
+  --button-primary-padding: 12px 23px;
+  --button-thanks-padding: 5px;
```
Comment (miketaylr) [inline webcompat/static/css/src/variables.css]: Not sure if `--button-secondary-padding` is better here? Or is `-thanks-` OK?

**Comment by zoepage** [comment]
@miketaylr feel free to review. gonna fix the rebase issue first thing tomorrow.

**Comment by miketaylr** [comment]
@zoepage LGTM! feel free to merge once you sort out the rebase stuff.

**Comment by zoepage** [comment]
As it looks ok locally, but github is displaying the history in a weird way I'll close this one and merge #2278

**Pull request closed without merging**